pydoctor/astbuilder.py

```diff
@@ -843,9 +868,11 @@ def visit_Assign(self, node: ast.Assign) -> None:
                 continue
             else:
                 if not isTupleAssignment:
+                    self._handleDocComment(node, target)
```

Comment:
I am slightly suspicious that these two functions seem to be always called together. Should they be tied together more closely?